v1.0/Specification.md

```diff
@@ -342,6 +342,9 @@ Table with terms used in the specification.
 | **Embedded objects** | An object embedded in the file/package i.e. an image, another spreadsheet or 3D object. |
 | **External cell references** | External cell references is a type of cell formula, which fetches data from another spreadsheet. |
 | **External objects** | An object that is linked as an OLE object, but is not embedded into the file/package. |
+| **File format** | A file format defines the data structure, scope of content and encoding of bits for any type of file. The file format may be standardised, it may contain other formats and it may be binary or text-based. The purpose of file format validation is two-fold: Checking if the file's purported format is correctly encoded and checking if any data quality rules, which delimits the file format's scope, are violated. |
```

Comment:
I would leave file format at this:
A file format defines the data structure, content scope and encoding of bits for a file. A file format may be standardised, contain other formats, and be binary or text-based.

This is a definition of format validation and should be listed as such:

The purpose of file format validation is two-fold: Checking if the file's purported format is correctly encoded and checking if any data quality rules, which delimit the file format's scope, are violated.

Reply:
I can adapt.